datasets/covid_qa_castorini/covid_qa_castorini.py

```diff
@@ -0,0 +1,117 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""CovidQA, a question answering dataset specifically designed for COVID-19."""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+import logging
+
+import datasets
+
+
+_CITATION = """\
+@article{tang2020rapidly,
+  title={Rapidly Bootstrapping a Question Answering Dataset for COVID-19},
+  author={Tang, Raphael and Nogueira, Rodrigo and Zhang, Edwin and Gupta, Nikhil and Cam, Phuong and Cho, Kyunghyun and Lin, Jimmy},
+  journal={arXiv preprint arXiv:2004.11339},
+  year={2020}
+}
+"""
+
+_DESCRIPTION = """\
+CovidQA is the beginnings of a question answering dataset specifically designed for COVID-19, built by hand from \
+knowledge gathered from Kaggle's COVID-19 Open Research Dataset Challenge.
+"""
+
+_HOMEPAGE = "http://covidqa.ai"
+
+_LICENSE = "Apache License 2.0"
+
+_URL = "https://raw.githubusercontent.com/castorini/pygaggle/master/data/"
+_URLs = {"covid_qa_castorini": _URL + "kaggle-lit-review-0.2.json"}
+
+
+class CovidQA(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class CovidQaCastorini(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `covid_qa_castorini`